src/js/lib/monster.util.js

```diff
@@ -976,6 +965,26 @@ define(function(require) {
 	}
 	util.formatPrice = formatPrice;
 
+	/**
+	 * Takes a string and replace all the "_" from it with a " ".
+	 * Also capitalizes first word.
+	 * Useful to display hardcoded data from the database that hasn't make it to
+	 * the i18n files.
+	 * @param  {*} variable Value to format.
+	 * @return {String} Formatted string representation of the value.
+	 */
+	function formatVariableToDisplay(variable) {
+		return _
+			.chain(variable)
+			.toString()
+			.replace(/_/g, ' ')
+			.replace(/\w\S*/g, function(txt) {
+				return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+			})
```

Comment:
Could `lodash#capitalize` be used instead?
```suggestion
			.replace(/\w\S*/g, _.capitalize)
```

Reply:
Sure, that works too!